frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tsx

```diff
@@ -55,34 +55,40 @@ const PoliciesTable = ({
 }: IPoliciesTableProps): JSX.Element => {
   const { config } = useContext(AppContext);
 
-  const emptyState = () => {
-    const emptyPolicies: IEmptyTableProps = {
-      graphicName: "empty-policies",
-      header: "You don't have any policies",
-      info:
-        "Add policies to detect device health issues and trigger automations.",
-    };
-    if (canAddOrDeletePolicy) {
-      emptyPolicies.primaryButton = (
-        <Button
-          variant="brand"
-          className={`${baseClass}__select-policy-button`}
-          onClick={onAddPolicyClick}
-        >
-          Add policy
-        </Button>
-      );
-    }
-    if (searchQuery) {
-      delete emptyPolicies.graphicName;
-      delete emptyPolicies.primaryButton;
-      emptyPolicies.header = "No matching policies";
-      emptyPolicies.info = "No policies match the current filters.";
-    }
-
-    return emptyPolicies;
+  const emptyState: IEmptyTableProps = {
+    graphicName: "empty-policies",
+    header: "You don't have any policies",
+    info:
+      "Add policies to detect device health issues and trigger automations.",
   };
 
+  if (currentTeam?.id === null || currentTeam?.id === -1) {
+    emptyState.header += " that apply to all teams";
+  } else {
+    emptyState.header += " that apply to this team";
+  }
+
+  if (canAddOrDeletePolicy) {
+    emptyState.primaryButton = (
+      <Button
+        variant="brand"
+        className={`${baseClass}__select-policy-button`}
+        onClick={onAddPolicyClick}
+      >
+        Add policy
+      </Button>
+    );
+  } else {
+    emptyState.info = "";
+  }
```

Comment:
Functional changes here:
* add suffix to main message depending on whether All Teams or a specific team is selected
* remove verbiage about adding policies if you don't have the ability to do so